README.md

```diff
@@ -155,9 +163,10 @@ such as restricting discovered features with the --label-whitelist option._
 
 ### Network Features
 
-| Feature name   | Description                                                                         |
-| :------------: | :---------------------------------------------------------------------------------: |
-| [SRIOV][sriov] | Single Root Input/Output Virtualization (SR-IOV) enabled Network Interface Card
+| Feature name     | Description                                                                         |
```

Comment:
Maybe it would be good to add a column for component name or sub-feature name? So for example. Feature Name, Component Name, Description? Might be confusing the users what the significance of the . is

Reply:
Looked at the readme rendered in browser, and I agree. It looked somehow confusing. I created a new column named Attribute